### PR TITLE
#18 Maya v1.0.0 Edge Evaluator

### PR DESCRIPTION
**Pull request by luisalbaceteinbibo**
### Description of this issue
Edge evaluator node documented.

### Pages to review
- `edge_evaluator.md`

Issue of this PR: #18

### CODE REVIEW

File: docs/documentation/maya/edge_evaluator.md [hunk 1]
```diff
@@ -1,34 +1,29 @@
-# How To Use
+# AdnEdgeEvaluator
 
-The Edge Evaluator Node is a dependency node to compute deformation changes in edges. Based on two input meshes will output a compression map of the edges.
+The AdnEdgeEvaluator node is a dependency node that computes deformation changes in edges. Based on two input meshes it will output a compression map of the edges.
```
Comment (lucasferryinbibo) [inline docs/documentation/maya/edge_evaluator.md:3]: We should maybe add an image with the result of the edhe evaluator and maybe an example of how to use or connect it to out deformers

**Review by lucasferryinbibo — CHANGES_REQUESTED** [review]
One small comment on my end

File: docs/documentation/maya/edge_evaluator.md (hunk 1 above)
Comment (lucasferryinbibo) [reply docs/documentation/maya/edge_evaluator.md:3]: I don't have the resources at the moment to provide that screenshot. I'm approving this PR given its current status

**Review by lucasferryinbibo — APPROVED** [review]

File: docs/documentation/maya/edge_evaluator.md [hunk 2]
```diff
@@ -1,34 +1,29 @@
...
 
 ## Requirements
 
 The Edge Evaluator Node requires the following inputs to be provided:
 
-  - <b class="mesh_color"> Rest Mesh (R) </b> with no deformation or animation.
-  - <b class="mesh_color"> Deform Mesh (D)</b> with deformations in its geometry. This deformations are not generated by an animation of a rig.
+  - <b class="mesh_color"> Rest Mesh (R):</b> Mesh with no deformation or animation.
+  - <b class="mesh_color"> Deform Mesh (D):</b> Mesh with deformations in its geometry. These deformations are not generated by the animation of a rig.
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/edge_evaluator.md:10]:
```suggestion
  - **Rest Mesh (R)**: Mesh with no deformation or animation.
  - **Deform Mesh (D)**: Mesh with deformations in its geometry. These deformations are not generated by the animation of a rig.

```

**Review by josevalverdeinbibo — APPROVED** [review]
Great job! Just one comment to check.

File: docs/documentation/maya/edge_evaluator.md (hunk 2 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/edge_evaluator.md]: Done.

**Review by carlosmonteagudoinbibo — APPROVED** [review]

**Pull request merged**